packages/url_launcher/url_launcher_web/lib/src/link.dart

```diff
@@ -142,13 +146,25 @@ class LinkViewController extends PlatformViewController {
 
   static int? _hitTestedViewId;
 
-  static late StreamSubscription<html.MouseEvent> _clickSubscription;
+  static final JSFunction _jsGlobalKeydownListener = _onGlobalKeydown.toJS;
+  static final JSFunction _jsGlobalClickListener = _onGlobalClick.toJS;
+
+  static void _onGlobalKeydown(html.KeyboardEvent event) {
+    // The keydown event is not directly associated with the target Link, so
+    // we need to look for the recently hit tested Link to handle the event.
+    if (_hitTestedViewId != null) {
```

Comment:
How does this `_hitTestedViewId` get set when using the keyboard?

Reply:
I tried to explain the flow in [this comment](https://github.com/flutter/packages/pull/6505#discussion_r1562659976). Please let me know if it's still unclear.